src/navigation/routesNames.ts

```diff
@@ -72,7 +72,6 @@ const Routes = {
   SEND_SHEET_NAVIGATOR: 'SendSheetNavigator',
   SETTINGS_SHEET: 'SettingsSheet',
   SHOWCASE_SHEET: 'ShowcaseSheet',
-  SIGN_TRANSACTION_SHEET: 'SignTransactionSheet',
```

Comment:
I'm seeing `Routes.SIGN_TRANSACTION_SHEET` still used in `ProfileActionButtonsRow.tsx`